### PR TITLE
log errors by default

### PR DESCRIPTION
**Pull request by ethangodt**
This PR is a suggestion to log errors all of the time.

Use `debug()` for extraneous logs that are only sometimes helpful. Logs about errors that occur are always helpful to the user and should not be swallowed except for in rare cases.

More thoughts:
- users shouldn't have to remember extra environment variables
- failing silently doesn't indicate an error and is a frustrating user experience (especially on CI where it is sometimes non-trivial to change job configuration)

### CODE REVIEW

File: lib/instance.js [hunk 1]
```diff
@@ -68,7 +66,7 @@ module.exports = function instance(input, done, progress) {
       try {
         require(module);
       } catch (err) {
-        throw new Error(err);
```
Comment (ethangodt) [inline lib/instance.js]: I don't know enough about the entire codebase to know whether we really want to throw here, or just log like in every other case. If we need to just throw then we can delete the try/catch altogether.

**Comment by ethangodt** [comment]
It's possible that we don't need to log in all of these cases. That said, I want to strongly recommend logging by default in any case that would bomb out the overall process. Running the command and getting back no response is deeply frustrating and worth avoiding (especially for anyone new that doesn't want to memorize the readme/codebase).

File: lib/instance.js (hunk 1 above)
Comment (grawk) [reply lib/instance.js]: interesting that we are catching/throwing ... already seems like the try/catch is extraneous.

**Comment by ethangodt** [comment]
updated the changelog

**Pull request merged**